styles/style.css

```diff
@@ -0,0 +1,448 @@
+.body {
+  font-family: 'Gotham Pro', sans-serif;
+  margin: unset;
+  font-size: 0.75em;
+}
+.body > * {padding: 0 calc(50vw - 8.75rem)}
+
+.header__title, .product__header, .product__price, .info__header, .stories__header, .story__header {font-size: 1.666em;}
+.header__title {line-height: .95em;}
+
+.header__arrow {
+  font-size: 3.3em;
+  line-height: .22em;
+}
+
+.product__name, .story__time, .footer {font-size: 1.25em;}
+
+.info__text, .story__text {font-size: 1.166em}
+
+.photos__header {font-size: 2.08em;}
+
+.footer__text {font-size: 1.08em;}
+
+.header {color: #fff;}
+
+h1, h2, h3, h4 {margin: unset;}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+button {
+  font: inherit;
+  background: inherit;
+  border: unset;
+  padding: unset;
+  text-transform: inherit;
+  cursor: pointer;
+}
+label {cursor: pointer;}
+button:hover, a:hover, label:hover {
+  box-shadow: 0 12px 16px -16px red,0 17px 50px -50px red;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)